src/sidebar/components/SelectionTabs.js

```diff
@@ -158,14 +158,14 @@ function SelectionTabs({ annotationsService, isLoading, settings }) {
         </div>
       )}
       {!isLoading && showNotesUnavailableMessage && (
-        <Frame classes="u-text--centered">
+        <Frame classes="text-center">
```

Comment:
...and some `text-` prefixed classes are static, like this one! This is a `text-align` rule.